src/main/groovy/de/dkfz/roddy/tools/TimeUnit.groovy

```diff
@@ -151,4 +151,24 @@ class TimeUnit {
     String toString() {
         return timeString
     }
+
+    String alignToHours() {
+        String[] split = timeString.split("[:]")
```

Comment:
This is parsing code with checks. When the parsing fails to only way to get information about what goes wrong is available via the debugger. Please add checks for unexpected conditions and throwing an Exception (at least) with the mis-parsed string and additional information, what assumption was made and not fulfilled.